src/shim.c

```diff
@@ -939,6 +940,15 @@ connect_to_proxy(struct cc_shim *shim)
 	return false;
 }
 
+/*
+ * Print version information.
+ */
+void
+show_version(void) {
+	printf("%s version: %s\n", PACKAGE_NAME, PACKAGE_VERSION);
+	printf("commit: %s\n", GIT_COMMIT);
```

Comment:
We want to make `cc-runtime cc-env` display this value (see https://github.com/clearcontainers/runtime/issues/238), but it's just occurred to me that it would be better if both the proxy and the shim version strings were 1 line only to support that. Alternatively, the runtime could just swallow any newlines.

@dlespiau - what do you think?